### ISSUE TITLE
Fix 40N6 engaging ballistic targets at very far ranges and altitudes + update max target speed

### ISSUE THREAD

**Issue by GIGANTO-pithecus** [PygmalionOfCyprus/cmo-db-requests#3431]
### DB Selector

DB3K

### Affected DBID(s)

2104

### Summary of Changes

Currently in game the 40N6 from sa-21a/b facility automatically launches towards ballistic targets at up to 40nm away in distance and at altitudes of 120,000-150,000 feet ASL. 
Exporter lists 60km maximum distance against ballistic targets (~31 nm) and a max engagement altitude of 25km/82,000ft ASL
Also, max target speed in game for 40N6 missile is set to 4500kts (6.8 mach) but on same site the exporter catalogue states 4800 m/s (mach 14) as max target speed

Here is a screenshot showing parameters listed by the export/manufacture catalogue:
![Screenshot (121)](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/134186902/1733d38f-070f-45f3-8373-762480550914)

Here is screenshot in game of 40n6 from battery automatically engaging short range ballistic missiles from a distance of nearly 40nm (75km) at an altitude of 5213 feet (below altitude minimum of 2km) I tested against scarab, fateh, himars gmlrs, same result - very long range engagement at ranges and altitudes exceeding what exporter lists on site.
![Screenshot (120)](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/134186902/c11122a4-2556-4139-bca7-1e19e526c01f)

Summary of changes:
-set parameters for 40N6 automatic anti-ballistic missile engagements for all ballistic target types to targets with (max 31nm in distance and max altitude of 25km/13.5nm)
-set max target speed to 9334.

### Sources

https://roe.ru/eng/catalog/air-defence-systems/air-defense-systems-and-mounts/s-400-triumf/

**Comment by HVAA** [comment]
I do see an issue with your target in the screenshot. Regardless of what the WRA says, 429kts at 4500' AGL is hardly a ballistic missile engagement and is closer to aerodynamic fligth characteristics.  I do agree that the parameters need looking at for the 40N6 as it is barely an ABM as it has no terminal manouevring to speak of, a J band seeker and a B/F warhead that may be optimised, but is not ideal for ABM (HtK is the way of things these days).  Far to capable against most targets when it's true target set is long range shots at HVAA and bombers.

**Comment by GIGANTO-pithecus** [comment]
> I do see an issue with your target in the screenshot. Regardless of what the WRA says, 429kts at 4500' AGL is hardly a ballistic missile engagement and is closer to aerodynamic flight characteristics. I do agree that the parameters need looking at for the 40N6 as it is barely an ABM as it has no terminal manouevring to speak of, a J band seeker and a B/F warhead that may be optimised, but is not ideal for ABM (HtK is the way of things these days). Far to capable against most targets when it's true target set is long range shots at HVAA and bombers.

It's just that based off specs its engaging targets like scud, scarab, iskander at super long ranges and either deflects or overshoots at higher altitude (and in some cases undershoots) due to outside the radius of engagement. It's high speed of mach 5-6 should allow it to be ABM but only within 25-30nm distance and as long as targets are beneath 82,000 feet ASL.

**Comment by HVAA** [comment]
Nothing in S-400 is HTK.  All missiles have a Prox Fused Blast Frag warhead. Catalogues are one thing, the exporter is trying to 'big up' the system.  Reality is another thing and it seems S-400 is having a hard time with HIMARS which they claim it should be able to shoot down, but things still keep being hit by HIMARS.

**Comment by GIGANTO-pithecus** [comment]
> Nothing in S-400 is HTK. All missiles have a Prox Fused Blast Frag warhead. Catalogues are one thing, the exporter is trying to 'big up' the system. Reality is another thing and it seems S-400 is having a hard time with HIMARS which they claim it should be able to shoot down, but things still keep being hit by HIMARS.

yes, but only confirmed s400 deployments are in crimea far from frontlines beyond reach of himars. S-300PS, PM2 and buk are dealing with himars mostly near frontlines.

**Comment by TTZ32** [comment]
Lol who the hell would waste expensive missiles from the S400 complex to intercept a guided mlrs rocket? Pantsir and Buk do a great job of dropping Himars rockets and theres plenty of video proof of that. Not to mention they are getting heavily jammed now.

**Comment by GIGANTO-pithecus** [comment]
> Lol who the hell would waste expensive missiles from the S400 complex to intercept a guided mlrs rocket? Pantsir and Buk do a great job of dropping Himars rockets and theres plenty of video proof of that. Not to mention they are getting heavily jammed now.

that wasn't the issue being addressed. it was the 70-90km engagement ranges against all types of ballistic missiles from the 40N6 which would overshoot or undershoot the target greatly, when in reality these ballistic missiles would only be engaged within 50km range at a max altitude 25km by the 40N6 for optimal probability of hit. Anything greater than that significantly reduces chances of impact or significant overshooting since missile climbs very fast.
9M96D series missiles of S-400 are also capable of ABM interceptions at ranges of 30km and altitudes of 25km for max target speeds up to mach 6. 

BUK-M1-2 has max target speed of mach 4 (range of 45km) so it must be positioned very close to defended target in order to reliably intercept an BM (plus 50%  chance of interception) whereas BUK-M3 has max target speed of mach 8.75 and range of 75km but improved probability of hit against ballistic missiles of 70%+ both systems would need to be a maximum of 10 km from defended point to ensure reliable hits. 

In the case of pantsir, its ABM role is limited greatly by range but capable of interceptions of targets up to mach 5. For reliable interception it would need to be placed extremely close to defended area to ensure any interceptions (as was shown in the video where it engaged 8 HIMARS rockets uploaded a while ago on youtube). 

Whole point was your system needs to be very close regardless of range to ensure max reliability in downing ballistic missiles. This doesn't apply however to THAAD, ARROW,  or S-500 as they have 150-600km interception ranges against any ballistic missile target and very long range tracking radars.
